app/templates/posts.hbs

```diff
@@ -7,6 +7,7 @@
     </header>
     <div class="gh-contentfilter">
         <div class="gh-contentfilter-left">
+            {{#unless session.user.isContributor}}
```

Comment:
We're hiding the post type dropdown here but from what I can see Contributor's are still able to create pages. Should we either a) not allow Contributors to create pages or b) allow them to filter between posts and pages?

Admittedly this is a bit of an edge case as it's unlikely a contributor will have enough draft posts and pages for the filter to be useful, I think the distinction of a contributor being allowed to create pages is the more important design decision.